packages/base/src/commands.ts

```diff
@@ -661,6 +673,7 @@ export function addCommands(
     },
     execute: args => {
       const current = tracker.currentWidget;
+      console.log('REDO PRESSED')
```

Comment:
I noticed `redo` is not working properly for me locally, I add a model -> I do some changes -> I undo them -> I try to redo, but it is not functioning properly.

I verified the same on [https://jupytercad.github.io/JupyterCAD/lab/index.html](https://jupytercad.github.io/JupyterCAD/lab/index.html) too.

I'd be happy to open an Issue for the same upon confirmation.

Reply:
Cheers, I already that branch locally too.
Thanks a lot @trungleduc 🎉 💯